kategory/src/main/kotlin/kategory/data/ListKW.kt

```diff
@@ -1,28 +1,9 @@
 package kategory
 
-@higherkind data class ListKW<A> constructor(val list: List<A>) : ListKWKind<A> {
-
-    fun <B> map(f: (A) -> B): ListKW<B> = ListKW(list.map(f))
-
-    fun <B> flatMap(f: (A) -> ListKW<B>): ListKW<B> = ListKW(list.flatMap { f(it).list })
-
-    operator fun plus(list: List<A>): ListKW<A> = ListKW(this.list + list)
-
-    operator fun plus(listKW: ListKW<A>): ListKW<A> = ListKW(this.list + listKW.list)
-
-    operator fun get(position: Int): Option<A> = if (list.isEmpty() || position < 0 || position > list.size) Option.None else Option.Some(list[position])
-
-    fun <B> fold(b: B, f: (B, A) -> B): B = list.fold(b, f)
-
-    fun drop(n: Int): ListKW<A> = ListKW(this.list.drop(n))
-
-    fun first(): A = this.list.first()
+@higherkind data class ListKW<A> constructor(val list: List<A>) : ListKWKind<A>, List<A> by list {
```

Comment:
Yes, until we have full wrapper generation with the annotation processor yin code gen phase. 
Since List is already Iterable all of those functions should be already available.